src/services/api.js

```diff
@@ -4,4 +4,42 @@ const api = axios.create({
   baseURL: 'https://api.github.com',
```

Comment:
Change the address to use OpenWeather API https://openweathermap.org/current

Reply:
But you already changed this URL, #4 . I didn't change to not have conflits

Reply:
This PR was closed

Reply:
Yeah, i saw that in the master already changed as doc the OpenWeather.
https://github.com/lucasmontano/openweathermap-reactnative/blob/ca3ee8d1385ab48f082454626e7cca06f64e60d9/src/services/api.js#L6